presto-hive/src/main/java/com/facebook/presto/hive/HiveLocationService.java

```diff
@@ -26,21 +28,24 @@
 
 import static com.facebook.presto.hive.HiveErrorCode.HIVE_FILESYSTEM_ERROR;
 import static com.facebook.presto.hive.HiveErrorCode.HIVE_PATH_ALREADY_EXISTS;
-import static com.facebook.presto.hive.HiveWriteUtils.createTemporaryPath;
```

Comment:
is `HiveWriteUtils.createTemporaryPath` used anywhere else? Maybe it is eligible to be removed now or replaced with the method you just added?